.github/workflows/release.yml

```diff
@@ -54,5 +55,7 @@ jobs:
           version: pnpm run version
           publish: pnpm run publish
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # Note: we are using a different GITHUB_TOKEN due to the issues in this thread:
+          # https://github.com/changesets/action/issues/187
+          GITHUB_TOKEN: ${{ secrets.CHANGESETS_GITHUB_TOKEN }}
```

Comment:
Looks great!
For testing sake, let's add a changeset to this PR (run pnpm changeset)
We don't necessarily have to publish it, we just want to see if it creates the PR and runs the checks.

Reply:
added this for testing: [.changeset/breezy-berries-fix.md](../blob/chore/version-packages-pr-fix/.changeset/breezy-berries-fix.md) 🤞